README.md

```diff
@@ -84,7 +82,7 @@ See [variables.tf] and [examples/] for details and use-cases.
 
   Default is `{}`.
 
-- [**`module_depends_on`**](#var-module_depends_on): *(Optional `list(any)`)*<a name="var-module_depends_on"></a>
+- [**`module_depends_on`**](#var-module_depends_on): *(Optional `list(object)`)*<a name="var-module_depends_on"></a>
```

Comment:
that is the wrong type.. please update from template

Reply:
why is any converted to object?

Reply:
Terradoc used to accept any as a type, but doesnt anymore

Reply:
In newer documentation the type is `list(dependency)`